bundles/mapping/infobox/resources/scss/infobox.ol.scss

```diff
@@ -127,20 +127,39 @@ div.popupHeader {
     margin: 0px;
     @include border-radius(6px 6px 0px 0px);
     display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    width: 390px;
```

Comment:
Can you test/provide some screenshots how this looks with a wide infobox? A long title or the content shown on the infobox might make the infobox wider than this. Does this need a set width?

Reply:
Actually these were not actually added even tho git seems to think they are, only the flex direction is changed. But on that note the width needs to be removed from the class "popupHeaderTitle". The width on this popupHeader class seems to be overwritten by something at least on my local environment (so it might not be needed anyways). I can provide couple more screenshots.

Reply:
[Line 565](https://github.com/oskariorg/oskari-frontend/blob/cb979b49bef1fc2197a7061fc14ea100f419d07b/bundles/mapping/infobox/plugin/openlayerspopup/OpenlayersPopupPlugin.ol.js#L565) this seems to (always?) make the width '100%' so should the width be removed from the css or make it '100%', or might there be something that 390px starting point is for?

Reply:
Oh, true. Most instances probably use the "adaptable: true" config for this. Well if the width isn't new addition I think it can stay there.